.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
   "editor.formatOnSave": true,
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
Why did this change

Reply:
probably when the file was saved, my vscode preferences changed this